src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -13,6 +13,7 @@
 
     public class ScenarioRunner
     {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Code", "PS0023:Use DateTime.UtcNow or DateTimeOffset.UtcNow", Justification = "Test logging")]
```

Comment:
Should we minimise the scope of suppressions? E.g. there is only one line in this method which triggers the diagnostic, so should we suppress it for that line only? Otherwise this could mask other, invalid usages of `Now`.

Reply:
2 lines, and it doesn't do much else.

Reply:
I'd _prefer_ to use `#pragma` here, for consistency with all the other `PS0023` suppressions, but I don't want to hold up the PR on that. After all, we already have a mix of `SuppressMessage` and `#pragma` for other PSXXXX suppressions.